packages/exhibitor/extra/dcos-exhibitor.service

```diff
@@ -7,11 +7,14 @@ StandardError=journal
 Restart=always
 StartLimitInterval=0
 RestartSec=5
+# Run in new mount namespace to create custom resolv.conf.
 MountFlags=private
+PermissionsStartOnly=true
 RuntimeDirectory=dcos_exhibitor
 EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/dns_config
 EnvironmentFile=/opt/mesosphere/etc/exhibitor
 EnvironmentFile=-/opt/mesosphere/etc/exhibitor-extras
-# run in new mount namespace to create custom resolv.conf
+# Change owner of realm file and ignore exit code, because file my not exist.
+ExecStartPre=/bin/sh -c "/usr/bin/chown dcos_exhibitor /opt/mesosphere/etc/exhibitor_realm || true"
```

Comment:
Agree. I have removed the chown and instead use more open file permissions downstream.